Tests/SwiftIfConfigTest/ActiveRegionTests.swift

```diff
@@ -0,0 +1,100 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2024 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+import SwiftDiagnostics
+import SwiftIfConfig
+import SwiftParser
+import SwiftSyntax
+import SwiftSyntaxMacrosGenericTestSupport
+import XCTest
+
+public class ActiveRegionTests: XCTestCase {
+  let linuxBuildConfig = TestingBuildConfiguration(
+    customConditions: ["DEBUG", "ASSERTS"],
+    features: ["ParameterPacks"],
+    attributes: ["available"]
+  )
+
+  func testActiveRegions() throws {
+    try assertActiveCode(
+      """
+      4️⃣
+      #if DEBUG
+      0️⃣func f()
+      #elseif ASSERTS
+      1️⃣func g()
+
+      #if compiler(>=8.0)
+      2️⃣func h()
+      #else
+      3️⃣var i
+      #endif
+      #endif
+      5️⃣token
```

Comment:
Should we rename these markers so that 4 is between 3 and 5 instead of before 0? Also, in the parser tests, we usually start with 1 instead of 0, it would be good to be consistent here as well.

Same in the other tests

Reply:
This is too nitpicky for me when there's substantive stuff to do.